CMake/External_qtExtensions.cmake

```diff
@@ -0,0 +1,35 @@
+# Qt
+add_package_dependency(
+  PACKAGE qtExtensions
+  PACKAGE_DEPENDENCY Qt
+  PACKAGE_DEPENDENCY_ALIAS Qt4
```

Comment:
Your code here raises a thought ... will this package build against Qt5? If it won't, it would be wise to check Qt_SELECT_VERSION for NOT VERSION_GREATER 4.8.6 or similar logic, and fail to configure if it's not the right version.

Reply:
@mwoehlke-kitware Will this build against Qt5?

Reply:
It will not currently build against Qt5, but there is a branch from @sankhesh to make it build against Qt5 that is currently under review.